samples/contrib/aws-samples/README.md

```diff
@@ -0,0 +1,198 @@
+        ]
+      }
+      EOF
+      ```
+   4. Create an IAM role using trust.json. Make a note of the ARN returned in the output.
+      ```
+      aws iam create-role --role-name kfp-example-pod-role --assume-role-policy-document file://trust.json
+      aws iam attach-role-policy --role-name kfp-example-pod-role --policy-arn arn:aws:iam::aws:policy/AmazonSageMakerFullAccess
+      aws iam get-role --role-name kfp-example-pod-role --output text --query 'Role.Arn'
+      ```
+   5. Edit your pipeline-runner service account.
+      ```
+      kubectl edit -n kubeflow serviceaccount pipeline-runner
+      ```
+      Add `eks.amazonaws.com/role-arn: <role_arn>` to annotations, then save the file. Example: **(add only line 5)**  
+      ```
+      apiVersion: v1
+      kind: ServiceAccount
+      metadata:
+        annotations:
+          eks.amazonaws.com/role-arn: <role_arn>
+          kubectl.kubernetes.io/last-applied-configuration: |
+            {"apiVersion":"v1","kind":"ServiceAccount","metadata":{"annotations":{},"labels":{"app":"pipeline-runner","app.kubernetes.io/component":"pipelines-runner","app.kubernetes.io/instance":"pipelines-runner-0.2.0","app.kubernetes.io/managed-by":"kfctl","app.kubernetes.io/name":"pipelines-runner","app.kubernetes.io/part-of":"kubeflow","app.kubernetes.io/version":"0.2.0"},"name":"pipeline-runner","namespace":"kubeflow"}}
+        creationTimestamp: "2020-04-16T05:48:06Z"
+        labels:
+          app: pipeline-runner
+          app.kubernetes.io/component: pipelines-runner
+          app.kubernetes.io/instance: pipelines-runner-0.2.0
+          app.kubernetes.io/managed-by: kfctl
+          app.kubernetes.io/name: pipelines-runner
+          app.kubernetes.io/part-of: kubeflow
+          app.kubernetes.io/version: 0.2.0
+        name: pipeline-runner
+        namespace: kubeflow
+        resourceVersion: "11787"
+        selfLink: /api/v1/namespaces/kubeflow/serviceaccounts/pipeline-runner
+        uid: d86234bd-7fa5-11ea-a8f2-02934be6dc88
+      secrets:
+      - name: pipeline-runner-token-dkjrk
+      ``` 
+**option 2]** Store the IAM credentials as a `aws-secret` in kubernetes cluster. Then use those in the components.
+   1. You need credentials for an IAM user with SageMakerFullAccess. Apply them to k8s cluster.
+      Replace `AWS_ACCESS_KEY_IN_BASE64` and `AWS_SECRET_ACCESS_IN_BASE64`.
+      > Note: To get base64 string you can do `echo -n $AWS_ACCESS_KEY_ID | base64`
+      ```
+      cat <<EOF | kubectl apply -f -
+      apiVersion: v1
+      kind: Secret
+      metadata:
+        name: aws-secret
+        namespace: kubeflow
+      type: Opaque
+      data:
+        AWS_ACCESS_KEY_ID: <AWS_ACCESS_KEY_IN_BASE64>
+        AWS_SECRET_ACCESS_KEY: <AWS_SECRET_ACCESS_IN_BASE64>
+      EOF
+      ```
+   2. Use the stored `aws-secret` in pipeline code by adding this line to each component in your pipeline `.apply(use_aws_secret('aws-secret', 'AWS_ACCESS_KEY_ID', 'AWS_SECRET_ACCESS_KEY'))`  
```

Comment:
I think this line does a good enough job of summarising how to adapt the sample pipelines to use hard-coded metrics. Lets use this as a reference rather than leaving the `.apply()` commented in every sample. It looks quite messy, and a little confusing at first sight.